src/js/components/account/user-registration.js

```diff
@@ -84,11 +86,18 @@ export class UserRegistration {
       const $formElementsContainer = $("<div />");
       $formElementsContainer.append(getLabel(field.label));
 
-      if (field.label === "password") {
+      if (field.label.toLowerCase() === "password") {
```

Comment:
Let's stop using presentation things for logic. Let's keep presentation and logic separate.

```suggestion
      if (field.name === "password") {
```